### PR TITLE
Leave a group chat

### PR DESCRIPTION
**Pull request by shamisem**

### CODE REVIEW

File: docs/en/using-status/leave-a-group-chat.md [hunk 1]
```diff
@@ -8,7 +8,31 @@ hide:
...
+If someone is adding you to group chats you don't want to be in, consider [changing who can invite you to group chats][change-who-can-invite-you-to-group-chats] or [blocking that person][block-someone-in-status].
+
+!!! note
+	The group chat owner can't leave without deleting the chat for everyone. For more information, check out [Delete a group chat][delete-a-group-chat].
+
+## Leave a group chat
+
+=== "Mobile"
+
+    1. From the tab bar, tap :mobile-messages: Messages.
```
Comment (jorge-campo) [inline docs/en/using-status/leave-a-group-chat.md:24]: This final **Messages** in step (1) goes in bold typeface, even when the text doesn't appear on the screen.

File: docs/en/using-status/leave-a-group-chat.md [hunk 2]
```diff
@@ -8,7 +8,31 @@ hide:
...
+    2. Open the group chat you want to leave.
+    3. Tap :mobile-more-options: **More options**.
+    4. Select :mobile-placeholder: **Leave group**.
+    5. Tap **Leave**.
+
+=== "Desktop"
+
+    1. From the navigation bar, click :desktop-chat: Chat.
```
Comment (jorge-campo) [inline docs/en/using-status/leave-a-group-chat.md:32]: This final **Chat** in step (1) goes in bold typeface.

File: docs/en/using-status/leave-a-group-chat.md [hunk 3]
```diff
@@ -8,7 +8,31 @@ hide:
 
 # Leave a group chat
 
- :octicons-tools-24:
+If you don’t want to be a member of a [group chat][understand-group-chats] anymore, you can leave the chat. You can always [re-join the chat][join-a-group-chat] later if you change your mind.
 
-!!! note ""
-     We're working on this content.
+Once you leave, a note appears in the group chat's message history letting other members know that you left the chat. You can still see the previous message history for a group chat you left.
```
Comment (jorge-campo) [inline docs/en/using-status/leave-a-group-chat.md:13]: > Once you leave, a note appears in the group chat's message history letting other members know that you left the chat.

This needs a screenshot to show the user what s/he can expect others to see. Could you please add the screenshot to [the Notion list](https://www.notion.so/Simplified-screenshots-in-Status-Help-9b99304babb042e2a63f4bc94aa0b179)?

Comment (jorge-campo) [inline docs/en/using-status/leave-a-group-chat.md:13]: > You can still see the previous message history for a group chat you left.

I'm sorry, I don't understand this. Do I still see the group chat, can I enter it and check my previous messages after leaving the chat?

File: docs/en/using-status/leave-a-group-chat.md [hunk 4]
```diff
@@ -8,7 +8,31 @@ hide:
...
+	The group chat owner can't leave without deleting the chat for everyone. For more information, check out [Delete a group chat][delete-a-group-chat].
+
+## Leave a group chat
+
+=== "Mobile"
+
+    1. From the tab bar, tap :mobile-messages: Messages.
+    2. Open the group chat you want to leave.
+    3. Tap :mobile-more-options: **More options**.
+    4. Select :mobile-placeholder: **Leave group**.
```
Comment (jorge-campo) [inline docs/en/using-status/leave-a-group-chat.md:27]: The mobile icons set in Figma contains this icon, no need for placeholders. It's the same icon we use for the user log-out action. I include the icon here with the correct dimensions (you still need to change the `fill`SVG property):

log-out

File: docs/en/using-status/leave-a-group-chat.md (hunk 2 above)
Comment (jorge-campo) [inline docs/en/using-status/leave-a-group-chat.md:32]: "navigation **side**bar" (see Style Guide for reference.)

**Review by jorge-campo — CHANGES_REQUESTED** [review]
Thanks, @shamisem ! 

Only minor changes from my side. As I mention inline, I don't understand what happens when I leave the group. If you can clarify this point and fix the other minor things, it's good to go.

File: docs/en/using-status/leave-a-group-chat.md (hunk 3 above)
Comment (shamisem) [reply docs/en/using-status/leave-a-group-chat.md:13]: Done! I created [an issue](https://github.com/status-im/help.status.im/issues/396) for us to remember about this once the design team finishes the screen.

**Pull request merged**